### PR TITLE
SCAL-101403: Added firefox specific css

### PR DESCRIPTION
**Pull request by aditya-kumar44**

### CODE REVIEW

File: docs/src/assets/styles/index.scss [hunk 1]
```diff
@@ -89,6 +89,12 @@ a {
     padding-left: 6px;
 }
 
+@-moz-document url-prefix() {
+    .documentBody {
+        padding-left: 2px;
+    }
+  }
+
```
Comment (kailash-choudhary-thoughtspot) [inline docs/src/assets/styles/index.scss:97]: why its happening for Mozilla only?

Comment (kailash-choudhary-thoughtspot) [reply docs/src/assets/styles/index.scss:97]: can you add comment why its specific to mozilla

Comment (aditya-kumar44) [reply docs/src/assets/styles/index.scss:97]: This is very common problem. CSS behave differently on different browsers. 
To fix we use below:

Mozilla uses -moz-
Chrome/Opera/Safari use -webkit-
Microsoft uses -ms-

Comment (akashmohapatra) [reply docs/src/assets/styles/index.scss:97]: Please double check the calculation of the left nav width as that seems to be different across both the browsers

**Review by akashmohapatra — CHANGES_REQUESTED** [review]

File: docs/src/assets/styles/index.scss (hunk 1 above)
Comment (aditya-kumar44) [reply docs/src/assets/styles/index.scss:97]: Problem seems to be from this line of code:
const { width, ref } = useResizeDetector();
In chorme it is giving width 1361 and in firefox it is giving 1054.
Due to this there is inconsistency between both the browsers.
Also, 
const [leftNavWidth, setLeftNavWidth] = useState(
        width > MAX_TABLET_RESOLUTION
            ? LEFT_NAV_WIDTH_DESKTOP
            : LEFT_NAV_WIDTH_TABLET,
    );

Here, leftNavWidth is never taking initial value. I checked after removing initial value, still it is taking 304 and 309. It is not being set from anywhere.

**Comment by aditya-kumar44** [comment]
Closing

**Pull request closed without merging**